### PR TITLE
Testing branch

### PR DESCRIPTION
**Pull request by Umuhan**
### Below are the changes made to the code:

- [x] Used jest framework for testing.

- [x] Created a test file ([..].test.js) for a file containing the add item and delete item functions.

- [x] Used Mock to test if add/delete functions 

- [x] Used the describe() method.

### CODE REVIEW

File: src/tests/addTask.test.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    expect(typeof addTask).toEqual('function');
+  });
+
+  test('should add a task to the tasks array', () => {
+    const tasks = [];
+    const description = 'Test task';
+    addTask(description);
+
+    expect(tasks).toHaveLength(0);
+  });
```
Comment (mikethreels) [inline src/tests/addTask.test.js]: - [ ] After adding a task to an empty task array the length of the array should be 1 instead of 0.

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- good job setting up the tests.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/tests/addTask.test.js (hunk 1 above)
Comment (Umuhan) [reply src/tests/addTask.test.js]: Ok. Thanks.
changes done.

**Review by TadesseDev — APPROVED** [review]
### Hi @Umuhan 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
AmazingCreepyFrogmouth-max-1mb




## Highlights
 - [x] Good job making the changes requested by the previous code reviewer. ✅


## Optional suggestions

- N/A


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**